### PR TITLE
Fhac-463: added compiler source, target entry to pom.xml

### PR DESCRIPTION
**Pull request by sailajaa**
1. Added compiler source, target entry to pom.xml

### CODE REVIEW

**Comment by nareshcgi** [comment]
Reviewed the PR and looks good.

File: pom.xml [hunk 1]
```diff
@@ -201,6 +213,7 @@
                     <outputFileNameMapping>@{artifactId}@-@{baseVersion}@.@{extension}@</outputFileNameMapping>
                 </configuration>
             </plugin>
+
```
Comment (christophermay07) [inline pom.xml]: Any reason for adding this blank line?

File: pom.xml [hunk 2]
```diff
@@ -193,6 +193,18 @@
                     </execution>
                 </executions>
             </plugin>
+			<plugin>
```
Comment (christophermay07) [inline pom.xml]: Convert tabs to spaces here

File: pom.xml [hunk 3]
```diff
@@ -193,6 +193,18 @@
...
+                <inherited>true</inherited>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.0</version>
```
Comment (christophermay07) [inline pom.xml]: Latest compiler plugin is 3.3

Comment (sailajaa) [inline pom.xml]: Leaving version as is, as we have 3.0 in rest of the pom's.  We can update to latest plugin in all pom files

**Pull request closed without merging**